server/methods/core/cart.js

```diff
@@ -434,6 +434,10 @@ Meteor.methods({
     // we need to get the parent of the option to check if parcel info is stored there
     const immediateAncestors = variant.ancestors.filter((ancestor) => ancestor !== product._id);
     const immediateAncestor = Collections.Products.findOne({ _id: immediateAncestors[0] });
+    // Get default parcel size
+    const shop = Collections.Shops.findOne({
+      _id: Reaction.getShopId
```

Comment:
should `getShopId` be a function there? Has this code ever been run? Doesn't look like it has.